rpcs3/Emu/NP/np_structs_extra.cpp

```diff
@@ -0,0 +1,211 @@
+		sceNp2.warning("flagFilter: 0x%x", req->flagFilter);
+		sceNp2.warning("flagAttr: 0x%x", req->flagAttr);
+		sceNp2.warning("intFilter: *0x%x", req->intFilter);
+		sceNp2.warning("intFilterNum: %d", req->intFilterNum);
+		sceNp2.warning("binFilter: *0x%x", req->binFilter);
+		sceNp2.warning("binFilterNum: %d", req->binFilterNum);
+		sceNp2.warning("attrId: *0x%x", req->attrId);
+		sceNp2.warning("attrIdNum: %d", req->attrIdNum);
+	}
+
+	void print_room_member_data_internal(const SceNpMatching2RoomMemberDataInternal* member)
+	{
+		sceNp2.warning("SceNpMatching2RoomMemberDataInternal:");
+		sceNp2.warning("next: *0x%x", member->next);
+		sceNp2.warning("npId: %s", member->userInfo.npId.handle.data);
+		sceNp2.warning("onlineName: %s", member->userInfo.onlineName->data);
+		sceNp2.warning("avatarUrl: %s", member->userInfo.avatarUrl->data);
+		sceNp2.warning("joinDate: %lld", member->joinDate.tick);
+		sceNp2.warning("memberId: %d", member->memberId);
+		sceNp2.warning("teamId: %d", member->teamId);
+		sceNp2.warning("roomGroup: *0x%x", member->roomGroup);
+		sceNp2.warning("natType: %d", member->natType);
+		sceNp2.warning("flagAttr: 0x%x", member->flagAttr);
+		sceNp2.warning("roomMemberBinAttrInternal: *0x%x", member->roomMemberBinAttrInternal);
+		sceNp2.warning("roomMemberBinAttrInternalNum: %d", member->roomMemberBinAttrInternalNum);
+		for (u32 i = 0; i < member->roomMemberBinAttrInternalNum; i++)
+			print_member_bin_attr_internal(&member->roomMemberBinAttrInternal[i]);
+	}
+
+	void print_room_data_internal(const SceNpMatching2RoomDataInternal* room)
+	{
+		sceNp2.warning("SceNpMatching2RoomDataInternal:");
+		sceNp2.warning("serverId: %d", room->serverId);
+		sceNp2.warning("worldId: %d", room->worldId);
+		sceNp2.warning("lobbyId: %lld", room->lobbyId);
+		sceNp2.warning("roomId: %lld", room->roomId);
+		sceNp2.warning("passwordSlotMask: %lld", room->passwordSlotMask);
+		sceNp2.warning("maxSlot: %d", room->maxSlot);
+
+		sceNp2.warning("members: *0x%x", room->memberList.members);
+		print_room_member_data_internal(room->memberList.members.get_ptr());
+		sceNp2.warning("membersNum: %d", room->memberList.membersNum);
+		sceNp2.warning("me: *0x%x", room->memberList.me);
+		sceNp2.warning("owner: *0x%x", room->memberList.owner);
+
+		sceNp2.warning("roomGroup: *0x%x", room->roomGroup);
+		sceNp2.warning("roomGroupNum: %d", room->roomGroupNum);
+		sceNp2.warning("flagAttr: 0x%x", room->flagAttr);
+		sceNp2.warning("roomBinAttrInternal: *0x%x", room->roomBinAttrInternal);
+		sceNp2.warning("roomBinAttrInternalNum: %d", room->roomBinAttrInternalNum);
+		for (u32 i = 0; i < room->roomBinAttrInternalNum; i++)
+			print_bin_attr_internal(&room->roomBinAttrInternal[i]);
+	}
+
+	void print_create_room_resp(const SceNpMatching2CreateJoinRoomResponse *resp)
+	{
+		sceNp2.warning("SceNpMatching2CreateJoinRoomResponse:");
+		sceNp2.warning("roomDataInternal: *0x%x", resp->roomDataInternal);
+		if(resp->roomDataInternal)
```

Comment:
```suggestion
		if (resp->roomDataInternal)
```

Reply:
Ah accidentally commited np_structs_extra, going to remove it